wimm.Secundatives.UnitTests/Extensions/VariantMap_Test.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+using wimm.Secundatives.Extensions;
+
+namespace wimm.Secundatives.UnitTests.Extensions
+{
+    public class VariantMap_Test
+    {
+        [Fact]
+        public void MapValue_ValueInt_ReturnsIntFuncResult()
+        {
+            var underTest = ConstructInt();
+            Assert.Equal(43, underTest.MapValue((x) => x + 1, x => x.Length));
```

Comment:
inconsistent parentheticization in the lambdas. not gonna block, just pointing it out